### PR TITLE
Add `raising-non-exception` example

### PR DESCRIPTION
**Pull request by harupy**
## Type of Changes

|     | Type                   |
| --- | ---------------------- |
| ✓   | :scroll: Docs          |

## Description

Refs #5953

Closes #XXXX

### CODE REVIEW

File: doc/data/messages/r/raising-non-exception/good.py [hunk 1]
```diff
@@ -1 +1 @@
-# This is a placeholder for correct code for this message.
+raise Exception
```
Comment (Pierre-Sassoulas) [inline doc/data/messages/r/raising-non-exception/good.py:1]:
```suggestion
raise Exception("Goodbye world !")
```

**Comment by Pierre-Sassoulas** [comment]
~~I think we should have valid code in good.py~~ Nvm did not know it's possible to raise without a message. It's bad style though, what do you think ?

**Comment by harupy** [comment]
> ~I think we should have valid code in good.py~ Nvm did not know it's possible to raise without a message. It's bad style though, what do you think ?

`raise Excpetion` is equivalent to `raise Exception`.

https://docs.python.org/3/tutorial/errors.html#raising-exceptions explains as follows:

> The sole argument to [raise](https://docs.python.org/3/reference/simple_stmts.html#raise) indicates the exception to be raised. This must be either an exception instance or an exception class (a class that derives from [Exception](https://docs.python.org/3/library/exceptions.html#Exception)). If an exception class is passed, it will be implicitly instantiated by calling its constructor with no arguments:

**Review by Pierre-Sassoulas — APPROVED** [review]
Thank you for teaching me something 😉

**Pull request merged**